app/views/layouts/application.html.erb

```diff
@@ -42,7 +42,9 @@
   <body>
     <%= render HeaderComponent.new %>
     <%# Toasts can be rendered into this container. %>
-    <div class="toast-container position-fixed top-0 end-0 p-3" style="z-index: 1080" id="toast-container"></div>
+    <div class="toast-container position-fixed top-0 end-0 p-3" style="z-index: 1080" id="toast-container">
```

Comment:
@justinlittman should I add this line to the ToastComponent? Or add a ToastWrapperComponent?